components/resources/BUILD.gn

```diff
@@ -88,6 +88,7 @@ repack("resources") {
 
   if (!is_android && !is_ios) {
     deps += [
+      "//brave/browser/resources/brave_new_tab_page_refresh:generated_resources",
```

Comment:
I don't know why gn check didn't catch this, but this is a layering violation. Same with the sources entry below.